tools/github-labels/common-labels.csv

```diff
@@ -244,6 +244,7 @@ Label,duplicate,This issue is the duplicate of another issue,e4c288
 Label,feature-request,This issue requires a new behavior in the product in order be resolved.,eaa875

```

Comment:
While in here can you also remove the "auto-merge" label above?

Reply:
Thanks. Mostly just wanted to make sure we didn't add back the auto-merge label when you run the sync based on this list.